.travis.yml

```diff
@@ -73,11 +73,11 @@ script:
   # Run linters
   - ./gradlew check
   # Run espresso tests in the Backpack module ( this excludes snapshot tests)
-  - ./gradlew Backpack:connectedCheck
+  - ./gradlew -Pandroid.testInstrumentationRunnerArguments.notAnnotation=androidx.test.filters.FlakyTest Backpack:connectedCheck
```

Comment:
these arguments can be put in `build.gradle` under `defaultConfig` as 
```testInstrumentationRunnerArgument "notAnnotation", "androidx.test.filters.FlakyTest"```

Reply:
But that would change every execution, we only want to ignore those tests on CI